ext/yarp/extension.h

```diff
@@ -3,7 +3,7 @@
 
 #include <ruby.h>
 #include <ruby/encoding.h>
-#include "yarp.h"
+#include <yarp.h>
```

Comment:
Why this change? I had changed it to `"yarp.h"` so that it still worked for mirroring when this file was in a slightly different place